src/test/java/com/hocs/test/pages/complaints/ComplaintsTriageAndInvestigation.java

```diff
@@ -160,8 +160,10 @@ public void openTheSeriousAndMinorComplaintCategoryAccordion() {
     }
 
     public void selectIEDETClaimCategory(String category) {
+        setSessionVariable("complaintCategory").to(category);
         List<WebElementFacade> claimCategories = findAll("//span[text()='" + category + "']/parent::legend/following-sibling::div//label");
-        recordCaseData.checkRandomCheckboxFromList(claimCategories);
+        String selectedClaimCategory = checkRandomCheckboxFromList(claimCategories);
+        setSessionVariable("claimCategory").to(selectedClaimCategory);
```

Comment:
if instead of setting this session variable you used 
'recordCaseData.addHeadingAndValueRecord(selectedClaimCategory, complaintCategory);'
 you wouldnt need the extra step in the scenario that you have created